api/indexer.oas2.json

```diff
@@ -829,6 +849,16 @@
           "description": "\\[appidx\\] application index.",
           "type": "integer"
         },
+        "created-at-round": {
+          "description": "Round when this application was created.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "destroyed-at-round": {
```

Comment:
I know we've talked about it but I looked around and seems like we use "deleted" everywhere for application. Probably worth sticking to it.